web/cypress/e2e/group_1_onboarding/accessibility.spec.ts

```diff
@@ -0,0 +1,86 @@
+import "cypress-axe";
+import { setMobileViewport } from "../../support/helpers/helpers";
+import { onOnboardingPageStartingBusiness } from "../../support/page_objects/onboardingPageNew";
+
+describe("Automated Accessibilty Testing [feature] [all] [group1]", () => {
+  describe("Onboarding Desktop Accessibility Test", () => {
```

Comment:
When these tests run, Cypress will concatenate the describes and the individual test names something like this:
`Automated Accessibilty Testing [feature] [all] [group1] Onboarding Desktop Accessibility Test Accessibility for All Other Businesses`

To make this slightly more readable, I might rename this describe block.
```suggestion
  describe("Onboarding - Desktop", () => {
```